db/orm.py

```diff
@@ -0,0 +1,88 @@
+from pathlib import Path
+from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy.orm import Mapped
+from sqlalchemy.orm import mapped_column
+from sqlalchemy.orm import relationship
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import ForeignKey
+from api.request_status_enum import RequestStatusEnum
+import sqlalchemy
+from contextlib import contextmanager
+from typing import List
+from sqlalchemy import Integer, String, DateTime, create_engine, CheckConstraint
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class User(Base):
+    __tablename__ = "user"
+    id = mapped_column(Integer, primary_key=True)
+    email = mapped_column(String, nullable=False, unique=True)
+    uploads: Mapped[List["Upload"]] = relationship(
+        back_populates="user", cascade="all, delete-orphan"
+    )
+
+
+class Upload(Base):
```

Comment:
Missing some usefull methods here - set status, is finished, get uploads path.. stuff like that